lib/versioner/trashcan_test.go

```diff
@@ -120,3 +124,69 @@ func writeFile(t *testing.T, filesystem fs.Filesystem, name, content string) {
 		t.Fatal(n, len(content), err)
 	}
 }
+
+func TestTrashcanCleanOut(t *testing.T) {
+	cfg := config.FolderConfiguration{
+		FilesystemType: fs.FilesystemTypeBasic,
+		Path:           "testdata",
```

Comment:
There's some legacy stuff here with tests that operate in `testdata`, but for new tests we should use either `fs.FakeFS` or a tempdir from `t.TempDir()`